changelog/add-5882-tracks-for-deposit-settings

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: dev
+
+Add tracks events to deposit settings
```

Comment:
I think we should leave this empty so all the tracks changes fall under one changelog ([there's a generic entry in this PR](https://github.com/Automattic/woocommerce-payments/pull/6188/files)).

TIL that it's ok to leave the string blank after selecting `patch`, although the tool strongly encourages adding an entry!

Reply:
I've removed this changelog entry